qemu/tests/ept_test.py

```diff
@@ -0,0 +1,40 @@
+from avocado.utils import process, cpu
+from virttest import env_process
+
+
+def run(test, params, env):
+    """
+    ept test:
+    1) Turn off ept on host
+    2) Check if reading kvm_intel parameter crash host
+    3) Launch a guest
+    3) Check no error in guest
+    4) Restore env, turn on ept
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+
+    if cpu.get_cpu_vendor_name() != 'intel':
+        test.cancel("This test is supposed to run on Intel host")
+
+    unload_cmd = params["unload_cmd"]
+    load_cmd = params["load_cmd"]
+    test_cmd = params["test_cmd"]
+    process.system(unload_cmd)
+    process.system(load_cmd % "0")
```

Comment:
Should record the value of the ept option before doing the test and restore at the end of the test.

Reply:
Thanks, updated.